src/GitHub.Exports/Models/LocalRepositoryModel.cs

```diff
@@ -5,63 +5,40 @@
 using System.Linq;
 using GitHub.Primitives;
 using GitHub.UI;
-using GitHub.VisualStudio.Helpers;
 using GitHub.Services;
 
 namespace GitHub.Models
 {
+    /// <summary>
+    /// A local repository.
+    /// </summary>
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
-    public class SimpleRepositoryModel : NotificationAwareObject, ISimpleRepositoryModel, IEquatable<SimpleRepositoryModel>
+    public class LocalRepositoryModel : RepositoryModelBase, ILocalRepositoryModel, IEquatable<LocalRepositoryModel>
     {
-        public SimpleRepositoryModel(string name, UriString cloneUrl, string localPath = null)
+        public LocalRepositoryModel(string name, UriString cloneUrl, string localPath = null)
+            : base(name, cloneUrl)
         {
-            Name = name;
-            Owner = cloneUrl.Owner;
-            CloneUrl = cloneUrl;
             LocalPath = localPath;
             Icon = Octicon.repo;
         }
 
-        public SimpleRepositoryModel(UriString cloneUrl)
-        {
-            Name = cloneUrl.RepositoryName;
-            Owner = cloneUrl.Owner;
-            CloneUrl = cloneUrl;
-            Icon = Octicon.repo;
-        }
-
-        public SimpleRepositoryModel(string path)
+        public LocalRepositoryModel(string path)
+            : base(ExtractRepositoryName(path), ExtractCloneUrl(path))
```

Comment:
This will cause IO and git repo loading calls to happen twice every time we create one of these, and these calls happen a lot, so it would be good to avoid it. These calls cost more than the side effect of calling `PropertyChanged` when there's no subscribers.